doc/release-notes-1.3.2.rst

```diff
@@ -0,0 +1,53 @@
+.. _ncs_release_notes_132:
+
+|NCS| v1.3.2 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.2 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* Zephyr Bluetooth LE Controller is now Bluetooth Core Specification v5.1 qualified.
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.3.2**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+
+Supported modem firmware
+************************
+
+This version of the |NCS| has been tested with the following modem firmware for cellular IoT applications:
+
+* mfw_nrf9160_1.2.1
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+* Zephyr Bluetooth LE controller:
+
+  * Fixed invalid disconnect reason in the disconnect complete event.
+  * Changed the controller HCI version to Bluetooth Core Specification v5.1.
```

Comment:
This sounds confusing. Do you mean "Changed the controller HCI version to be compatible with/to follow the Bluetooth Core Specification v5.1."?
(Please don't change right now, I'm working on the branch ;) )

Reply:
I only changed the version number, and the version number means Bluetooth Core Specification v5.1

Reply:
Changed from where? This wasn't in the previous release notes.

I don't understand what we're trying to say here. How can the HCI version be the Bluetooth specification?
But if that's what we want to say, fine.

Reply:
https://www.bluetooth.com/specifications/assigned-numbers/link-layer/

In 1.3.0 and 1.3.1 it is 5.2. We also didn't include qualification before, so we haven't mentioned it's version either.

Reply:
I think it should be something like 
```suggestion
  * Selected the controller HCI version that corresponds to the Bluetooth Core Specification v5.1.
```
But if that's wrong, leave as is.

Reply:
This should be: "The controller HCI version, reported with the Read Local Version Information HCI command, has been changed to match version 5.1 of the Bluetooth Core Specification".

Reply:
```suggestion
  * Changed the controller HCI version (reported with the Read Local Version Information HCI command) to match version 5.1 of the Bluetooth Core Specification.
```